homeassistant/components/hive/climate.py

```diff
@@ -198,11 +224,23 @@ async def async_set_preset_mode(self, preset_mode):
             temperature = curtemp + 0.5
             await self.hive.heating.setBoostOn(self.device, 30, temperature)
 
-    @refresh_system
     async def async_heating_boost(self, time_period, temperature):
+        """Handle boost heating service call."""
+        _LOGGER.warning(
+            "Hive Service heating_boost will be deprecated in 2021.6.0 Please update to heating_boost_on"
```

Comment:
```suggestion
            "Hive Service heating_boost will be removed in 2021.6.0, please update to heating_boost_on"
```

Reply:
I'm assuming that we intend to remove it in 2021.6.0.

Reply:
@MartinHjelmare yep good spot and updated